xml_inscriptions/transcribed/NY.NY.CU.Butl.L.38.xml

```diff
@@ -0,0 +1,160 @@
+                     <placeName ref="europe.italy.rome">Italy, Rome, fuori Porta Salaria</placeName>
+                  </origin>
+                  <provenance type="transferred" subtype="sold">
+                     <p>Purchased by <persName type="buyer">George N. Olcott</persName> in
+                           <placeName ref="europe.italy.rome">Rome</placeName>
+                        <date when="1900">in the early 1900s</date>
+                     </p>
+                  </provenance>
+                  <provenance type="observed" subtype="reported">
+                     <p>Currently housed in the Rare Books and Manuscript Library in the <placeName>Columbia University Butler Library</placeName>
+                     </p>
+                  </provenance>     
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#AJA_Olcott1"/>
+                  <biblScope unit="item">158, no. 9</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CIL_VI"/>
+                  <biblScope unit="item">38943</biblScope>
+               </bibl>
+               <bibl>
+                  <ref type="external" target="http://www.edr-edr.it/edr_programmi/res_complex_comune.php?do=book&amp;id_nr=EDR114892">EDR114892</ref>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2024-09-28" who="Norah Mezey-McMahon">Added transcription and
+            translation, updated metadata</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here-->
+   <text>
+      <body>
+         <div type="edition">
+         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+         <ab> 
+            <lb n="1"/>
+               <persName> 
+                  <name type="gentilicium" key="Suetonía"> Suetonia</name>          
+               </persName>
+            <lb n="2"/>
+               <persName> 
+                  <name type="praenomen" key="Gaius">
+                     <expan><abbr>C</abbr><ex>aii</ex></expan>
+                  </name>
+                  <name type="praenomen" key="Gaius">
+                     <expan><abbr>C</abbr><ex>aii</ex></expan>
+                  </name>
+               </persName>
```

Comment:
These are presumably two separate people named Gaius, no? If so, we should have the `</persName>` after the end of the first abbreviation, and then maybe a comma to indicate that you're moving to another people. Alternatively, if we're understanding it as something like "C(aiorum) et C(aiae) liberta", you could mark this as `<persName><expan><abbr>C<am>C</am></abbr><ex>aiorum</ex></expan></persName>`. I'll defer to @jbodel 's interpretation here, but either way we'll want to tweak this very slightly.

Reply:
We want the former, that is, indicating each individual individually. So "Gaius and Gaius and a woman", not "two Gaii" or (worse) "two Gaiuses" (!). The interpretation should make clear that these letters represent individuals, not just names.